src/main/resources/db/migration/V42_8__Create_comment_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
when u will merge preprod, herizo have already put it so you have to delete that line after